pom.xml

```diff
@@ -902,6 +908,28 @@
             <autoVersionSubmodules>true</autoVersionSubmodules>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>findbugs-maven-plugin</artifactId>
+          <version>${findbugs.version}</version>
+          <dependencies>
+            <dependency>
+              <groupId>org.apache.beam</groupId>
+              <artifactId>java-build-tools</artifactId>
+              <version>${project.version}</version>
+            </dependency>
+          </dependencies>
+          <configuration>
+            <excludeFilterFile>beam/findbugs-filter.xml</excludeFilterFile>
+          </configuration>
+          <executions>
+            <execution>
+              <goals>
+                <goal>check</goal>
```

Comment:
This binds by default to the `verify` phase. If you aren't going to change the phase, do you have to specify it here? But actually, I prefer an explicit binding anyhow, and I also strongly prefer it to be as early in the lifecycle as possible, since it can function like a better type system. Is this the earliest it can be?